packages/bpk-component-input/src/BpkInput.js

```diff
@@ -128,12 +111,7 @@ class BpkInput extends Component<Props, State> {
       // $FlowFixMe[cannot-spread-inexact] - inexact rest. See 'decisions/flowfixme.md'.
       <input
         className={classNames.join(' ')}
-        ref={(input) => {
-          ref = input;
-          if (inputRef) {
-            inputRef(input);
-          }
-        }}
+        ref={forwardedRef}
```

Comment:
I guess we'd still need the component to work even if there is no ref passed as `inputRef` is an optional prop?

Reply:
I think with forward refs if you pass no ref it just handle it as null so its not the same as not sending a prop. Let me check

Reply:
Yea it just passes it as null - https://codesandbox.io/s/react-playground-forked-s7k7cd?file=/index.js